.github/CODEOWNERS

```diff
@@ -426,7 +426,7 @@
 /core/__init__.py @nithusha21
 /core/controllers/__init__.py @nithusha21
 /core/domain/__init__.py @nithusha21
-/core/domain/config*.py @DubeySandeep
+/core/domain/config*.py @DubeySandeep @BenHenning @vinitamurthi
```

Comment:
You can take @DubeySandeep off this one -- it now has adequate coverage.

Reply:
Done.